files/lodash.compat/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "bower",
+  "name": "lodash.compat",
+  "repo": "lodash/lodash",
+  "files": {
+    "include": ["**/*.js", "**/*.map"]
```

Comment:
I dont see any map files in there. I think a simple "include": ["lodash.min.js"] would be enough, you can even remove the unminified file from this PR.
What do you think?